tokio/benches/mpsc.rs

```diff
@@ -268,3 +247,37 @@ fn contended_bounded_tx(b: &mut Bencher) {
         th.join().unwrap();
     }
 }
+
+fn bench_mpsc(c: &mut Criterion) {
+    c.bench_function("bounded_new_medium", bounded_new_medium);
+    c.bench_function("unbounded_new_medium", unbounded_new_medium);
+    c.bench_function("bounded_new_large", bounded_new_large);
+    c.bench_function("unbounded_new_large", unbounded_new_large);
+    c.bench_function("send_one_message", send_one_message);
+    c.bench_function("send_one_message_large", send_one_message_large);
+    c.bench_function("bounded_rx_not_ready", bounded_rx_not_ready);
+    c.bench_function("bounded_tx_poll_ready", bounded_tx_poll_ready);
+    c.bench_function("bounded_tx_poll_not_ready", bounded_tx_poll_not_ready);
+    c.bench_function("unbounded_rx_not_ready", unbounded_rx_not_ready);
+    c.bench_function("unbounded_rx_not_ready_x5", unbounded_rx_not_ready_x5);
+    c.bench_function("bounded_uncontended_1", bounded_uncontended_1);
+    c.bench_function("bounded_uncontended_1_large", bounded_uncontended_1_large);
+    c.bench_function("bounded_uncontended_2", bounded_uncontended_2);
```

Comment:
Some of these benchmarks with different bounds seem like they could maybe be converted into parameterized benchmarks later. But, I think that can be a follow-up.